lib/controllers/patients/core.js

```diff
@@ -25,31 +27,31 @@ var initialFilterInput = crud.filterInputGenerator(initialKeys),
     returnListData = crud.returnListData;
 
 // Create new patient with current user given write access by default
-patients.post("/", auth.authenticate, initialFilterInput, function (req, res, next) {
+patients.post("/", auth.authenticate, guard(["admin"]), initialFilterInput, function (req, res, next) {
```

Comment:
Should this also include programAdmin?

Reply:
I think no, we aren't allowing any user to have more than one patient I believe. I only allowed "admin" to let the tests for this pass.